### PR TITLE
FeatureToggls: remove IsFeatureToggleEnabled from SettingsProvider

### PR DESCRIPTION
**Pull request by ryantxu**
This is a small cleanup after #44443 -- it removes `IsFeatureToggleEnabled` from settings provider *but* it still exists on settings.

Removing it from settings will require some more involved rejiggering because it is used within the settings package too.

### CODE REVIEW

File: pkg/cmd/grafana-cli/runner/wireexts_oss.go [hunk 1]
```diff
@@ -5,18 +5,22 @@ package runner
...
+	"github.com/grafana/grafana/pkg/services/licensing"
 	"github.com/grafana/grafana/pkg/services/sqlstore/migrations"
 	"github.com/grafana/grafana/pkg/setting"
 )
 
 var wireExtsSet = wire.NewSet(
 	wireSet,
 	migrations.ProvideOSSMigrations,
+	licensing.ProvideService,
+	wire.Bind(new(models.Licensing), new(*licensing.OSSLicensingService)),
```
Comment (papagian) [inline pkg/cmd/grafana-cli/runner/wireexts_oss.go:23]: Is this change related to the rest?

**Review by papagian — COMMENTED** [review]
The wire changes do not seem related, otherwise LGTM.

File: pkg/cmd/grafana-cli/runner/wireexts_oss.go (hunk 1 above)
Comment (ryantxu) [reply pkg/cmd/grafana-cli/runner/wireexts_oss.go:23]: yes -- required for the kmsproviders initialization

**Review by papagian — APPROVED** [review]
